desktop-environment.el

```diff
@@ -164,6 +164,23 @@ portion of the screen."
   :type 'string)
 
 
+
```

Comment:
please remove the blank line between the `^L` and the comment?

Reply:
I knew that there would be some rogue spaces in my version. :)